src/algorithm/structures/sensitive/data.rs

```diff
@@ -0,0 +1,57 @@
+use crate::response_code::{Error, Result, WrapperErrorKind};
+use crate::tss2_esys::TPM2B_SENSITIVE_DATA;
+use log::error;
+use std::convert::TryFrom;
+/// Rust native representation of sensitive data.
+///
+/// The structure contains the sensitive data as a byte vector.
+#[derive(Clone, Debug, Default)]
+pub struct SensitiveData {
```

Comment:
This structure seems to be a good candidate to derive [`Zeroize`](https://docs.rs/zeroize/1.1.0/zeroize/#custom-derive-support) on! This can probably be done in a later PR where we check this crate for structures that need zeroizing.